src/utils/wallet.js

```diff
@@ -274,14 +286,23 @@ export class Wallet {
    }
 
    requestCode(phoneNumber, accountId) {
-      return sendJson('POST', `${ACCOUNT_HELPER_URL}/account/${phoneNumber}/${accountId}/requestCode`)
+      return sendJson(
+         'POST',
+         `${ACCOUNT_HELPER_URL}/account/${phoneNumber}/${accountId}/requestCode`
+      )
    }
 
    async validateCode(phoneNumber, accountId, securityCode) {
```

Comment:
@marcinbodnar can you revert this and other formatting-only changes not related to your PR? It looks like was re-formatted automatically with settings which aren't very good in vertical space usage

Reply:
Done. Sorry. I can't turn off this weird function so I switch off code formatter.